gateway-ha/src/main/java/com/lyft/data/gateway/ha/router/ResourceGroupManager.java

```diff
@@ -0,0 +1,160 @@
+package com.lyft.data.gateway.ha.router;
+
+import com.lyft.data.gateway.ha.persistence.JdbcConnectionManager;
+import com.lyft.data.gateway.ha.persistence.dao.ResourceGroup;
+import com.lyft.data.gateway.ha.persistence.dao.Selector;
+import java.util.List;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class ResourceGroupManager implements PrestoResourceManager {
```

Comment:
It shold follow the same naming convention as the other files. It should be `HaResourceGroupsManager`. 

Ha* files are the implementations.